code/modules/mob/living/carbon/human/emote_vr.dm

```diff
@@ -25,6 +25,10 @@
 		if ("nme")
 			nme()
 			return TRUE
+		if("chirp")
```

Comment:
Please wrap the new function in //Yawn Edit and //End Yawn Edit  thanks.

Reply:
Sorry forgot, not use to coding downstream